uaclient/config.py

```diff
@@ -19,6 +19,8 @@ def cast(_, x):  # type: ignore
 
 
 DEFAULT_STATUS = {
+    "_doc": "Presence of keys services are considered Experimental and may"
```

Comment:
Given that we haven't specified this format at all, I think we can be more general and say that the entire output should not be considered stable.